repository/x.json

```diff
@@ -45,8 +45,9 @@
 			]
 		},
 		{
-			"name": "Xcode Build Configuration",
-			"details": "https://github.com/peterthomashorn/xcode-build-configuration-language-for-sublime-text",
+			"name": "Xcode Build Configuration and Project Syntax Definitions",
```

Comment:
Lol, ok, bit long for a name but that works. "Xcode Builds and Projects" maybe?

Reply:
Yeah, I know, but somehow I would like to keep it clear this is only about syntax definitions, not full support of such or anything else. 😅

Reply:
In case an additional syntax is added I will definitely think about the title again to make it shorter.